src/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Box, Button } from '@chakra-ui/react';
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
+
+import styles from '../css/Header.module.css';
+
+function Header(): JSX.Element {
+  const data: GatsbyTypes.HeaderQueryQuery = useStaticQuery<GatsbyTypes.HeaderQueryQuery>(graphql`
+    query HeaderQuery {
+      contentfulAsset(title: { eq: "Logo" }) {
+        fluid(quality: 100) {
+          ...GatsbyContentfulFluid
+        }
+        description
+      }
+    }
+  `);
+
+  return (
+    <div className={styles.navigation}>
+      <Box w="150px">
+        {data.contentfulAsset?.fluid != null ? (
+          <Img fluid={data.contentfulAsset.fluid} alt={data.contentfulAsset.description} />
+        ) : null}
+      </Box>
+      <input
```

Comment:
We'll have our own Chakra `Input` variant to work with but I haven't merged in the PR yet - feel free to just make this a Chakra `Input` for now